terminate_on_chain_service.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2023-2024 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
```

Comment:
I thought we had already such a script? O/w how did it work before on the non-staking side?

Reply:
The staking branch was incorporated in the main logic of run_service.sh. 
The thing is that 'run_service' is aimed at having a service running.
Therefore un-staking, terminating, unbonding  was immediately followed by 'update service', so that the service was in a running state.

What we want in this new use case is giving the user a mechanism to 'stop' the loop.

Reply:
We could probably remove some duplicate code between the two scripts in the future.

Reply:
Yes - let's open an issue to deduplicate this later

Reply:
Good point. Oppened an issue for this https://github.com/valory-xyz/trader-quickstart/issues/178